src/QsCompiler/DataStructures/Diagnostics.fs

```diff
@@ -712,14 +712,13 @@ type DiagnosticItem =
 
             | ErrorCode.ArrayBaseTypeMismatch -> "The array item type {0} does not match the expected type {1}."
             | ErrorCode.AmbiguousTypeParameterResolution ->
```

Comment:
Would it also make sense to add a compiler error documentation page, similar to https://docs.microsoft.com/azure/quantum/qs-compiler-messages/qs3308? IQ# automatically adds links to pages of that form when reporting errors, so it could help users resolve that.

Reply:
Yes, that could make sense, although it's not in my plans right now to write that page.